### PR TITLE
Change to omni output to include latency

### PR DESCRIPTION
**Pull request by jtaleric**
Instead of using regex to capture results, switch to the omni output. Moving to the omni output we can capture more results such as the latency values seen from netperf.

Todo : write latency values to dedicated csv

Signed-off-by: Joe Talerico (rook) <joe.talerico@gmail.com>

### CODE REVIEW

File: netperf.yml [hunk 1]
```diff
@@ -3,29 +3,29 @@ TCPStream:
    profile: "TCP_STREAM"
    duration: 10
    samples: 1
-   messagesize: 16384
+   messagesize: 1024
```
Comment (dceara) [inline netperf.yml:6]: Super nit: why do we need to change the input yml file in this PR?

File: netperf/kubernetes.go [hunk 2]
```diff
@@ -45,7 +45,7 @@ func GetZone(c *kubernetes.Clientset) (string, error) {
 	}
 	for _, l := range n.Items {
 		if len(l.GetLabels()["topology.kubernetes.io/zone"]) < 1 {
-			return "", fmt.Errorf("⚠️ No zone label")
+			return "", fmt.Errorf("⚠️  No zone label")
```
Comment (dceara) [inline netperf/kubernetes.go:48]: nit: Unrelated?

File: netperf/kubernetes.go [hunk 3]
```diff
@@ -57,7 +57,7 @@ func GetZone(c *kubernetes.Clientset) (string, error) {
 	}
 	// No zone had > 1, use the last zone.
 	if zone == "" {
-		fmt.Println("⚠️ Single node per zone")
+		fmt.Println("⚠️  Single node per zone")
```
Comment (dceara) [inline netperf/kubernetes.go:60]: nit: Unrelated?

File: netperf/kubernetes.go [hunk 4]
```diff
@@ -139,7 +139,7 @@ func GetPods(c *kubernetes.Clientset, dp DeploymentParams) (apiv1.PodList, error
 func CreateService(sp ServiceParams, client *kubernetes.Clientset) (*apiv1.Service, error) {
 	s, err := client.CoreV1().Services(sp.Namespace).Get(context.TODO(), sp.Name, metav1.GetOptions{})
 	if err == nil {
-		fmt.Println("♻️ Using existing Service")
+		fmt.Println("♻️  Using existing Service")
```
Comment (dceara) [inline netperf/kubernetes.go:142]: This too?

**Review by dceara — APPROVED** [review]
I only performed a superficial review and didn't test that this actually works fine.  With that said, the change looks good to me, thanks!

File: netperf.yml (hunk 1 above)
Comment (jtaleric) [reply netperf.yml:6]: @dceara yeah, so i noticed on my local testing that UDP would show super low throughput with the size being > MTU. So I reduced the size to < normal MTU.

File: netperf/kubernetes.go (hunk 2 above)
Comment (jtaleric) [reply netperf/kubernetes.go:48]: So, my OCD was flaring up, and without this extra space the Emoji/text wasn't aligning :D

File: netperf/kubernetes.go (hunk 3 above)
Comment (jtaleric) [reply netperf/kubernetes.go:60]: So, my OCD was flaring up, and without this extra space the Emoji/text wasn't aligning :D

File: netperf/kubernetes.go (hunk 4 above)
Comment (jtaleric) [reply netperf/kubernetes.go:142]: So, my OCD was flaring up, and without this extra space the Emoji/text wasn't aligning :D

Comment (dceara) [reply netperf/kubernetes.go:142]: I have a different type of OCD and would've moved this to a different commit.  But it's absolutely no big deal.  It's good to go as-is too.

File: netperf.yml (hunk 1 above)
Comment (dceara) [reply netperf.yml:6]: Ack.
